tasks/src/app/ordered-list/ordered-list.component.html

```diff
@@ -0,0 +1,8 @@
+<button (click)="changeListStatus()">{{ isListEnable ? "Disable" : "Enable" }}</button>
+<div *ngIf="isListEnable">
+	<ol>
+		<ng-container *ngFor="let name of listNames">
+			<li class="highlightingName" [ngClass]="{'highlightingName': name == 'Catalin'}"> {{name}} </li>
```

Comment:
```suggestion
			<li [ngClass]="{'highlightingName': name === 'Catalin'}"> {{name}} </li>
```

Reply:
you want that highlightingName to appear only if it is your name, not for each item of the list

Reply:
dubious but the "class" attribute was no longer taken into consideration when I used [ngClass] and everything went as required :))